tooling/windows_repro_process.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# shellcheck disable=SC2086
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -e
+
+JDK_DIR="$1"
+SELF_CERT_FILE="$2"
+SELF_CERT_PASS="$3"
+
+removeSignatures() {
+  echo "Removing Signatures from ${JDK_DIR}"
+  FILES=$(find "${JDK_DIR}" -type f -path '*.exe' && find "${JDK_DIR}" -type f -path '*.dll')
+  for f in $FILES
+    do
+      echo "Removing signature from $f"
+      if signtool remove /s $f ; then
+        echo "  ==> Successfully removed signature from $f"
+      else
+        echo "  ==> $f contains no signature"
+      fi
+    done
+  echo "Successfully removed all Signatures from ${JDK_DIR}"
+}
+
+# This script unpacks the JDK_DIR and removes windows signing Signatures in a neutral way
+# ensuring identical output once Signature is removed.
+
+if [[ ! -d "${JDK_DIR}" ]] || [[ ! -d "${JDK_DIR}/bin"  ]]; then
+  echo "$JDK_DIR does not exist or does not point at a JDK"
+  exit 1
+fi
+
+echo "Expanding the 'modules' Image to remove signatures from within.."
+jimage extract --dir "${JDK_DIR}/lib/modules_extracted" "${JDK_DIR}/lib/modules"
+rm ${JDK_DIR}/lib/modules
+
+echo "Expanding the 'src.zip' to normalize file permissions"
+unzip ${JDK_DIR}/lib/src.zip -d ${JDK_DIR}/lib/src_zip_expanded
```

Comment:
I'd personally be tempted to use `-q` on the `unzip` to avoid lots off stuff going to the console.